markdown/chapters/meandeck/brainstorm.md

```diff
@@ -26,48 +63,40 @@ Brainstorm post-Doomsday as long as you have an extra card in hand to put back.
+you to play around removal like {{Swords to Plowshares}}.
 
-> {B}{B}{B} + Brainstorm + X
-
-<row variant="pile">{{!LED}} {{!SW}} {{!IU}} {{!LP}} {{!TO}}</row>
+## Brainstorm Pile - Brainstorm and *Combo Piece* Available
 
-1. Cast Brainstorm and put back X then Ideas Unbound on top
-1. Cast LED, cycle SW cracking LED for {U}{U}{U} and draw the IU
-1. Cast IU, drawing the rest of the pile
-1. The Lotus Petal gives you a second {U} to cast TO with an empty library
+You may find yourself in a scenario where you have a card in hand that is
+considered *dead* outside of the combo turn. Thankfully, Brainstorm (BS) allows
+you to transfer it out of your hand and replace it into the pile. Let us
+consider the example above. It may be the opponent has a card like {{Scavenging
+Ooze}} out and you need to utilize LED to generate mana but don't want to
+discard your one Oracle to have it *eaten* by the Ooze. We cannot use Predict
+here as we would need to *mill* the Oracle to do so effectively. In many
+situations you might be able to make a PTT pile however if your life total is
+under pressure you may not be able to.
 
-## Brainstorm and *Combo Piece* Available
+Consider the following example:
 
 > {B}{B}{B} + Brainstorm + Thassa's Oracle
 
 <row variant="pile">{{!LED}} {{!SW}} {{!Pnd}} {{!EoA}} {{!Pnd}}</row>
 
-You may find yourself in a scenario where you have a card in hand that is
-considered *dead* outside of the combo turn. Thankfully, BS allows you to take
-it out of your hand and replace it into the pile. Let us consider the example
-above. It may be the opponent has a card like Scavenging Ooze out and you need
-to utilize LED to generate mana but don't want to discard your one Oracle to
-have it *eaten* by the Ooze. We cannot use Predict here as we would need to mill
-the Oracle to do so effectively.
-
 1. Cast Brainstorm and put back Oracle then Ponder on top.
 1. Cast LED, cycle SW cracking LED for {U}{U}{U} and draw the Ponder.
 1. Cast Ponder and arrange your library so that it is (EoA, TO, Pn)
 1. Draw Edge, Cycle Edge and draw Oracle.
 1. Cast Oracle to win.
 
-As you can see this pile does not beat removal on Oracle as it leaves a single
-card in your deck however, if you have an additional {U} available then you can
-change step 3 above to arrange the deck (EoA, Pn, TO) and cast the second Ponder
-with the additional {U}. Likewise, if you have another cantrip in addition to
-the Oracle in hand, you can use that instead of the first Ponder in the example
-and use that extra card to generate the additional {U} needed for the same
-starting mana cost.
+As you can see this is an imperfect pile as it does not beat removal on Oracle
+due to it leaving a single card in your deck. If you have an additional {U}
+available then you can change step 3. above to arrange the deck (EoA, Pn, TO)
+and cast the second Ponder with the additional {U}. Likewise, if you have
+another cantrip in addition to the Oracle in hand, you can use that instead of
```

Comment:
> another cantrip

Might need to mention that the cantrip needs to be able to re-arrange the 3 bottom cards before going through the pile rather than after like you did.

Reply:
Added